pallets/vc-management/src/lib.rs

```diff
@@ -96,6 +149,15 @@ pub mod pallet {
 		VCSubjectMismatch,
 		/// The VC is already disabled
 		VCAlreadyDisabled,
+
+		/// Error when the caller account is not the admin
+		RequireSchemaAdmin,
+		/// Schema Id already exists
```

Comment:
simply English issue, I think it's normally written like:
```
		/// Schema Id already exists
		SchemaAlreadyExist,
		/// Schema Id does not exist
		SchemaNotExist,
```

Reply:
Thanks! Fixed.